test/ui-test/testSuites/suite_status/tst_wallet/test.feature

```diff
@@ -10,44 +10,29 @@ Feature: Status Desktop Wallet
         When the user activates wallet and opens the wallet section
         When the user accepts the signing phrase
 
-   @mayfail
-   Scenario Outline: User adds a watch only account
-       When the user adds watch only account with <account_name> and <address>
-       Then the new account <account_name> is added
-
-       Examples:
-         | account_name | address                                      |
-         | one          | 0x8397bc3c5a60a1883174f722403d63a8833312b7   |
-         | two          | 0xf51ba8631618b9b3521ff4eb9adfd8a837455226   |
-
-
-    Scenario Outline: User generates a new account from wallet
-        When the user generates a new account with <account_name> and TesTEr16843/!@00
-        Then the new account <account_name> is added
-        When the user deletes the account <account_name>
-        Then the account <account_name> is not in the list of accounts
-
-        Examples:
-          | account_name |
-          | one          |
-
-   @mayfail
-   Scenario Outline: User imports a private key
-       When the user imports a private key with <account_name> and TesTEr16843/!@00 and <private_key>
-       Then the new account <account_name> is added
-
-       Examples:
-         | account_name | private_key |
-         | one          | 8da4ef21b864d2cc526dbdb2a120bd2874c36c9d0a1fb7f8c63d7f7a8b41de8f |
-
-   @mayfail
-   Scenario Outline: User imports a seed phrase
-       When the user imports a seed phrase with <account_name> and TesTEr16843/!@00 and <seed_phrase>
-       Then the new account <account_name> is added
-
-       Examples:
-         | account_name | seed_phrase |
-         | one          | indoor dish desk flag debris potato excuse depart ticket judge file exit |
+   Scenario: User can manage a list of accounts
```

Comment:
I'm thinking that maybe we can explore the `bdd hook` options and instead of merging scenarios, keeping simple scenarios, by using other methods that we are not using now, like `OnFeatureStart/OnFeatureEnd` or `OnStepStart/OnStepEnd`, for example, having the rule of just only starting or stoping the app in the OnFeatureStart / End so the app will be launched once and each scenario will start with the app in an idle state in the main window view. WDYT?

Take a look at section `6.19.10` of this link: https://doc.froglogic.com/squish/latest/api.bdt.functions.html

Reply:
Out of this PR, but just taking a chance to ask... 
@noeliaSD why don't we use the same account created using steps described in the `Background` section for all tests within the same test feature, but create new user each time? That will speed up a bit tests on CI and also give us opportunity to test some other features which require a user and accounts for it to be already added.

Reply:
> 

We do it in some tests, like the group chat test (don't know if it is still working), but what happen with this approach is that we don't have an easy way to cleanup the account so the way to have a fresh account is, creating a new one each time. I agree with you on it, so if we can manage the cleanup, we can try it. Ideally, we should have some test accounts and reuse them and a backend call like `reset account`, to cleanup the account to an idle/fresh state.

An example of cleanup we could need (to clarify): If we create a new community, channels, etc, then we need to ensure to cleanup it for the next tests iteration. And applicable for all actions and scenarios.

Reply:
> we don't have an easy way to cleanup the account

This is also a bug and that must be fixed.